<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WelcomeMenu</class>
 <widget class="QWidget" name="WelcomeMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>554</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pBtComponent">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>170</y>
     <width>341</width>
     <height>71</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Create Component</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pBtCircuit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>280</y>
     <width>341</width>
     <height>71</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Create Circuit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pBtCircuitText">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>390</y>
     <width>341</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Open Circuit File</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>451</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;h2 align=&quot;center&quot; style=&quot; margin-top:16px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:x-large; font-weight:600;&quot;&gt;Welcome to Circuit Design!&lt;/span&gt;&lt;/h2&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
